### PR TITLE
Update the readme with instructions for setting up webots

### PR DESCRIPTION
**Pull request by ysims**
Just links to NUbook :smile:

### CODE REVIEW

**Comment by tayloryoung6396** [comment]
Windows is pretty straight forward, use the linux install instruction for webots using the wsl. You'll also need 
In Windows, install VcXsrc from https://sourceforge.net/projects/vcxsrv/
In Windows, run XLaunch and ensure the following settings are set
First screen:
Multiple Windows
Display number: -1
Second screen:
Start no client
Third screen:
Clipboard: Checked
Primary Selection: Checked
Native opengl: Unchecked
Disable: Checked
In WSL, install needed xcb libraries
sudo apt install libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-render-util0 libxcb-xfixes0 libxcb-xinerama0 libxcb-xkb1 libxkbcommon-x11-0 libxkbcommon0
In Windows, open a command prompt and run ipconfig. Look for an adapter that does NOT have vEthernet in its name and has a valid IPv4 Address, note it down
In WSL set the DISPLAY environment variable
export DISPLAY=<IPv4 Address from step 4>:0.0 
You can add to this command to the end of your ~/.bashrc file, but the IP address is likely to change every time you reboot Windows so you need to remember to update this command and re-source ~/.bashrc
In WSL, source the ~/.bashrc file if you have edited it. If you have not changed ~/.bashrc since the last time you started WSL then you should not need to do this, but it also shouldn't hurt
source ~/.bashrc
At this point you should now be able to run graphical applications (like the labeller) in WSL.

**Comment by Bidski** [comment]
> Windows is pretty straight forward, use the linux install instruction for webots using the wsl. You'll also need
> In Windows, install VcXsrc from https://sourceforge.net/projects/vcxsrv/
> In Windows, run XLaunch and ensure the following settings are set
> First screen:
> Multiple Windows
> Display number: -1
> Second screen:
> Start no client
> Third screen:
> Clipboard: Checked
> Primary Selection: Checked
> Native opengl: Unchecked
> Disable: Checked
> In WSL, install needed xcb libraries
> sudo apt install libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-render-util0 libxcb-xfixes0 libxcb-xinerama0 libxcb-xkb1 libxkbcommon-x11-0 libxkbcommon0
> In Windows, open a command prompt and run ipconfig. Look for an adapter that does NOT have vEthernet in its name and has a valid IPv4 Address, note it down
> In WSL set the DISPLAY environment variable
> export DISPLAY=<IPv4 Address from step 4>:0.0
> You can add to this command to the end of your ~/.bashrc file, but the IP address is likely to change every time you reboot Windows so you need to remember to update this command and re-source ~/.bashrc
> In WSL, source the ~/.bashrc file if you have edited it. If you have not changed ~/.bashrc since the last time you started WSL then you should not need to do this, but it also shouldn't hurt
> source ~/.bashrc
> At this point you should now be able to run graphical applications (like the labeller) in WSL.

Also need to disable access control in VcXSrv settings

**Review by KipHamiltons — CHANGES_REQUESTED** [review]
Need to download/install webots first, before making the project

File: README.md [hunk 1]
```diff
@@ -4,44 +4,47 @@ This is the Webots world, models and controllers for simulating the NUbots robot
...
-```
+    ```sh
+    cmake ..
+    ```
 
-Navigate to the NUbots Webots folder
+    3. Compile the code
+    
+    ```sh
+    make
```
Comment (KipHamiltons) [inline README.md:35]:
```suggestion
    make -j$(nproc)
```

File: README.md [hunk 2]
```diff
@@ -4,44 +4,47 @@ This is the Webots world, models and controllers for simulating the NUbots robot
...
-| Folder      | Description                                                                                                        |
-| ----------- | ------------------------------------------------------------------------------------------------------------------ |
-| controllers | Controller files written in C++. These connect to a model, eg the robot model, and determine how it works.         |
-| protos      | Proto files, which define the models for objects.                                                                  |
-| worlds      | World files, which are opened in Webots and define what is in the environment and how the environment initialises. |
-
 ## Set Up
 
-For Ubuntu (tested on Linux Mint).
+On Windows, use Ubuntu in WSl. You can follow the instructions for setting it up on the [Getting Started NUbook page](https://nubook.nubots.net/guides/main/getting-started)
```
Comment (Bidski) [inline README.md:9]:
```suggestion
On Windows, use Ubuntu in WSL. You can follow the instructions for setting it up on the [Getting Started NUbook page](https://nubook.nubots.net/guides/main/getting-started)
```

File: README.md [hunk 3]
```diff
@@ -4,44 +4,47 @@ This is the Webots world, models and controllers for simulating the NUbots robot
...
+    3. Compile the code
+    
+    ```sh
+    make
+    ```
 
-```sh
-cd Webots
-```
+3. Download Webots from source using the instructions on [GitHub](https://github.com/cyberbotics/webots/wiki#installation-of-the-webots-development-environment).
```
Comment (Bidski) [inline README.md:38]:
```suggestion
3. Download and build Webots from source using the instructions on [GitHub](https://github.com/cyberbotics/webots/wiki#installation-of-the-webots-development-environment). 
```

**Review by Bidski — CHANGES_REQUESTED** [review]

**Review by KipHamiltons — APPROVED** [review]
Sure

**Review by Bidski — APPROVED** [review]
Sure, why not

**Pull request merged**